nextcloudappstore/core/templates/user/form-fields.html

```diff
@@ -0,0 +1,31 @@
+{% load css_class %}
+{% csrf_token %}
+{% if form.non_field_errors  %}
+    {% for error in form.non_field_errors %}
+        <span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true"></span>
+        <span class="sr-only">(error)</span>
+        <p class="text-danger">{{ error }}</p>
+    {% endfor %}
+{% endif %}
+{% for field in form %}
+    <div class="form-group {% if field.errors %}has-error has-feedback{% endif %}">
+        <label class="control-label" for="{{ field.id_for_label }}">{{ field.label }}</label>
+
+        {{ field|css_class:'form-control' }}
+
+        {% if field.errors %}
+            {% for error in field.errors %}
+                <span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true"></span>
+                <span class="sr-only">(error)</span>
+                <p class="text-danger">{{ error }}</p>
+            {% endfor %}
+        {% endif %}
+
+        {% if field.help_text %}
+            <p class="help-block">{{ field.help_text }}</p>
+        {% endif %}
+    </div>
+{% endfor %}
+{% if redirect_field_value %}
```

Comment:
Hi,

I guess we don't need it there.

I think it might makes sense to use that include in the our other forms as well though. If I remember correctly all we do in our other forms is loop over the form and style via css.